mlflow/models/container/__init__.py

```diff
@@ -142,8 +151,15 @@ def _serve_pyfunc(model):
     bash_cmds = []
     if pyfunc.ENV in conf:
         if not disable_env_creation:
-            _install_pyfunc_deps(MODEL_PATH, install_mlflow=True, enable_mlserver=enable_mlserver)
-        bash_cmds += ["source /miniconda/bin/activate custom_env"]
+            activate_cmd = _install_pyfunc_deps(
+                MODEL_PATH,
+                install_mlflow=True,
+                enable_mlserver=enable_mlserver,
+                env_manager=env_manager,
+            )
+        else:
+            activate_cmd = []
+        bash_cmds += activate_cmd
```

Comment:
This block needs to be fixed. When running the image, `disable_env_creation` is False so we fall into the else branch and `activate_cmd` is set to an empty array. If it's an empty array, the environment to serve the model is not activated when running the image.